libcaf_test/caf/test/runner.cpp

```diff
@@ -121,6 +123,18 @@ std::optional<std::regex> to_regex(std::string_view regex_string) {
 #endif
 }
 
+auto default_log_component_filter() {
+  using namespace std::literals;
+  return std::vector{
+    "caf"s,        // legacy CAF logging
+    "caf_flow"s,   // legacy CAF logging
+    "caf.core"s,   // events from log::core
+    "caf.io"s,     // events from log::io
+    "caf.net"s,    // events from log::net
+    "caf.openssl"s // events from log::openssl
```

Comment:
Should we add the additional `system` component here?

Reply:
No, `caf.system` is never suppressed by default anywhere. That component is only used if something goes really wrong, so we want to have that information available.